TMDBApp/TMDBApp/General.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Geral.swift
+//  TMDBApp
+//
+//  Created by Richard Leh on 19/12/2016.
+//  Copyright © 2016 Richard Leh. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SVProgressHUD
+
+typealias CompletionResultError = (_ result: Any?, _ error: Error?) -> Void
```

Comment:
I might be wrong but I'm not finding this typealias used anywhere else in the code.